tests/relay/test_rln.py

```diff
@@ -0,0 +1,22 @@
+import os
+import pytest
+from src.libs.custom_logger import get_custom_logger
+from src.steps.relay import StepsRelay
+
+logger = get_custom_logger(__name__)
+
+
+@pytest.mark.usefixtures()
+class TestRelayRLN(StepsRelay):
+    def test_register_rln(self):
+        logger.debug(f"Running register RLN test for main relay nodes")
+        key_stores_found = 0
+        rln_creds_source = "./rln_creds.json"
+
+        for k in range(1, 6):
+            self.register_rln_single_node(rln_creds_source=rln_creds_source, rln_creds_id=f"{k}")
+            keystore_path = f"/keystore_{k}/keystore.json"
+            if os.path.exists(keystore_path):
```

Comment:
wasn't check_rln_registration created to be used here?

Reply:
Used at https://github.com/waku-org/waku-interop-tests/pull/16/commits/0f593ccd1462f9da5ab8c23fb9ef846a65da43c6